Tests/AutoPilotTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AutoPilotTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import XCTest
+@testable import KeyboardSupport
+
+class AutoPilotTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var viewController: AutoNavigatorViewController! = nil
+    private var window: UIWindow!
+    
+    // MARK: - Tests
+    
+    override func setUp() {
+        super.setUp()
+        viewController = UIStoryboard.init(name: "Main", bundle: Bundle(for: AutoPilotTests.self)).instantiateViewController(withIdentifier: "AutoNavigatorViewController") as? AutoNavigatorViewController
+        
+        viewController.loadViewIfNeeded()
+        _ = viewController.view
```

Comment:
Accessing the view controller's view after calling `loadViewIfNeeded()` seems a bit redundant. Are you sure it's necessary?

Reply:
Since this test is dealing with a storyboard and views, we may want to consider creating UI tests also.

Reply:
Either line was required, but not both. This test is more about executing the view walking capability of the AutoPilotStatic methods. I could explore UI tests if we see a benefit to them. 

https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/f206620586ccb5fb8ffacb432087a02667f9ec92